### PR TITLE
Reformat navbar and footer

### PR DESCRIPTION
**Pull request by janakitti**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -77,7 +77,7 @@ const routes: Routes = [
 ];
 
 @NgModule({
-  imports: [ RouterModule.forRoot(routes, {scrollPositionRestoration: 'enabled'})], // scroll to top when routerLinking
+  imports: [ RouterModule.forRoot(routes)], // scroll to top when routerLinking
```
Comment (GuyInFridge) [inline src/app/app-routing.module.ts:80]:
```suggestion
  imports: [ RouterModule.forRoot(routes)], 
```
Remove comment

File: src/app/pages/contact/contact.component.html [hunk 2]
```diff
@@ -1,6 +1,21 @@
-<div class="container pt-3">
-  <h1 id="title">Contact Us</h1>
+<div class="container">
+  <h1 class="mt-3" id="title">Contact Us</h1>
   <br>
+<<<<<<< HEAD
```
Comment (GuyInFridge) [inline src/app/pages/contact/contact.component.html:4]: Uhhhhhhhhhhhh why is there a merge conflict in here lol.

Comment (GuyInFridge) [reply src/app/pages/contact/contact.component.html:4]: Seems like this contact us page is out of date with the one where it mentions our Facebook and Instagram (I think that one was on static-components branch? I'm not sure). Wouldn't want conflicting merges into master, so check if everything works together..

File: src/app/pages/faq/faq.component.html [hunk 3]
```diff
@@ -33,7 +33,11 @@ <h2 class="mb-0">
         <h2 class="mb-0">
           <button class="btn btn-link collapsed" type="button" data-toggle="collapse" data-target="#s1q2c"
                   aria-expanded="false" aria-controls="s1q2c">
+<<<<<<< HEAD
```
Comment (GuyInFridge) [inline src/app/pages/faq/faq.component.html:36]: Fix merge conflict.

File: src/app/pages/faq/faq.component.html [hunk 4]
```diff
@@ -54,7 +58,11 @@ <h2 class="mb-0">
         <h2 class="mb-0">
           <button class="btn btn-link collapsed" type="button" data-toggle="collapse" data-target="#s1q3c"
                   aria-expanded="false" aria-controls="s1q3c">
+<<<<<<< HEAD
```
Comment (GuyInFridge) [inline src/app/pages/faq/faq.component.html:61]: Fix merge conflict.

File: src/app/pages/contact/contact.component.html (hunk 2 above)
Comment (ricetech) [reply src/app/pages/contact/contact.component.html:4]:
```suggestion
```

File: src/app/pages/faq/faq.component.html (hunk 3 above)
Comment (ricetech) [reply src/app/pages/faq/faq.component.html:36]:
```suggestion
```

File: src/app/pages/faq/faq.component.html (hunk 4 above)
Comment (ricetech) [reply src/app/pages/faq/faq.component.html:61]:
```suggestion
```

File: src/app/core/footer/footer.component.html [hunk 5]
```diff
@@ -2,20 +2,19 @@
 <footer class="vertical-center align-items-center" id="footer">
   <div>
     <a class="m-4" href="https://www.facebook.com/virtrolio/" target="_blank">
-      <img src="../../../assets/images/icons/facebook.svg" alt="Our Facebook" width="32" height="32">
+      <img src="../../../assets/images/icons/facebook.svg" alt="Our Facebook" width="24" height="24">
```
Comment (TommyHuang-dev) [inline src/app/core/footer/footer.component.html:5]: Potentially missing units on the width and height, was this intended? The same problem occurs on the instagram and email icons as well.

File: src/assets/css/main.css [hunk 6]
```diff
@@ -11,8 +11,8 @@ html {
   --light-grey: #999999;
 
   /* Sizes */
-  --footer-height: 7em;
-  --navbar-height: 4.5em;
+  --footer-height: 5.5em;
+  --navbar-height: 3em;
```
Comment (ricetech) [inline src/assets/css/main.css:15]: Personally not a fan of the smaller navbar size - it makes the profile picture look out of place, and shrinking the picture would make it a bit too small imo
image

File: src/app/core/footer/footer.component.css [hunk 7]
```diff
@@ -4,6 +4,7 @@ footer {
   margin-bottom: 0;
   width: 100%;
   background-color: var(--darkest-accent);
+  font-size: 0.8em;
```
Comment (ricetech) [inline src/app/core/footer/footer.component.css:7]: This font size is a bit too small to be readable.

File: src/app/core/footer/footer.component.css [hunk 8]
```diff
@@ -22,3 +23,8 @@ footer > * {
 a {
   color:white;
 }
+
+.social-icon {
+  height: 1.5em;
+  width: 1.5em;
+}
```
Comment (ricetech) [inline src/app/core/footer/footer.component.css:30]: Same thing as the font, it seems too small.

File: src/assets/css/main.css (hunk 6 above)
Comment (GuyInFridge) [reply src/assets/css/main.css:15]: I know that Arvind wanted something which emulated a thinner navbar like Facebook's. I think **3.5em** for the navbar does that. The footer is fine, I think.

File: src/app/core/footer/footer.component.css (hunk 7 above)
Comment (GuyInFridge) [reply src/app/core/footer/footer.component.css:7]: Facebook uses **12px** and with 0.8em I'm seeing **13px** on my screen. I think the text is fine.

File: src/app/core/footer/footer.component.css (hunk 8 above)
Comment (GuyInFridge) [reply src/app/core/footer/footer.component.css:30]: I'd say try out **2em** (with appropriate margins to make it not look weird.

File: src/app/pages/msg-sent/msg-sent.component.html [hunk 9]
```diff
@@ -6,7 +6,7 @@ <h1 id="title-text">{{ name }} will receive your message soon!</h1>
     <!-- Paper plane SVG -->
     <img alt="Paper plane graphic" src="../../../assets/images/MessageSent-Light.svg" id="graphic">
   </div>
-  <div class="row justify-content-center mb-3" id="buttons">
+  <div class="row justify-content-center" id="buttons">
```
Comment (ricetech) [inline src/app/pages/msg-sent/msg-sent.component.html:9]:
```suggestion
  <div class="row justify-content-center mb-3" id="buttons">
```

File: src/app/core/footer/footer.component.html [hunk 10]
```diff
@@ -2,20 +2,20 @@
 <footer class="vertical-center align-items-center" id="footer">
```
Comment (ricetech) [inline src/app/core/footer/footer.component.html:2]: Add exclusions for `href`
```suggestion
<!--suppress HtmlUnknownTarget -->
<footer class="vertical-center align-items-center" id="footer">
```

**Pull request merged**